.travis.yml

```diff
@@ -9,7 +9,6 @@ services:
 
 env:
   matrix:
-    - IMAGE_NAME="ubuntu-upstart:14.04"
```

Comment:
Ubuntu 14.04 is no longer supported by PGDG, so PostgreSQL 12 doesn't exist.  It's quite old, so we should stop supporting it.

Reply:
I checked quickly (https://wiki.ubuntu.com/Releases). Standard support for 14.04 ended in April 2019, so this sounds reasonable to reduce the burden here!